src/core_plugins/region_map/public/region_map_vis_params.html

```diff
@@ -56,9 +67,12 @@
   </div>
 </div>
 
+
 <div class="kuiSideBarSection">
   <div class="kuiSideBarSectionTitle">
-    <div class="kuiSideBarSectionTitle__text">Basic settings</div>
+    <div class="kuiSideBarSectionTitle__text">
+      Base Layer Settings
+    </div>
   </div>
-  <vislib-basic-options></vislib-basic-options>
```

Comment:
was this intentional ? (removing the basic options ... so legend position and 'show tooltip' option ?)

Reply:
that's my mistake. wasn't too careful with rebase.